server/apps/publish/formatters/aap_ipnews_formatter.py

```diff
@@ -175,115 +128,3 @@ def format(self, article, subscriber):
 
     def can_format(self, format_type, article):
         return format_type == 'AAP IPNEWS' and article['type'] in ['text', 'preformatted']
```

Comment:
All formatters are based on the string literals at the moment. Out of curiosity I did a (global) search say 'preformatted' I've got 33 results so it is used in literal form pretty much every where. I suggest we discuss this approach and create a task to refactor the literals globally if we all agree.